app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt

```diff
@@ -63,8 +66,16 @@ class TabTrayViewHolder(
 
         // Drawables and theme
         updateBackgroundColor(isSelected2)
-        thumbnailView.setImageBitmap(tab.thumbnail)
-        iconView?.setImageBitmap(tab.icon)
+
+        if (tab.thumbnail != null) {
+            thumbnailView.setImageBitmap(tab.thumbnail)
+            thumbnailView.visibility = View.VISIBLE
+            iconView?.visibility = View.INVISIBLE
+        } else {
+            thumbnailView.visibility = View.INVISIBLE
+            iconView?.setImageBitmap(tab.icon)
+            iconView?.visibility = View.VISIBLE
```

Comment:
Lame nit but could we get these in the same order in both sides of the if/else?

Reply:
Fixed.